tests/test_api.py

```diff
@@ -194,6 +197,10 @@ def test_reporter(self):
     def test_reporter_with_branches(self):
         sh.coverage('run', '--branch', 'runtests.py')
         results = self.cover.get_coverage()
+
+        if self.oldCoverage and 'branches' not in results:
```

Comment:
I did not want to depend on a particular version of Python because e.g. alpha/beta versions of Python 3.6 or matching versions of PyPy3 could work differently.